<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="f974a6dd71ba" failures="1" tests="9" name="com.umesh.tests.vendorportal.VendorPortalTest" time="81.529" errors="0" timestamp="2024-08-29T16:10:43 GMT" skipped="1">
  <testcase classname="com.umesh.tests.vendorportal.VendorPortalTest" name="loginTest" time="14.053"/>
  <testcase classname="com.umesh.tests.vendorportal.VendorPortalTest" name="logoutTest" time="8.307"/>
  <testcase classname="com.umesh.tests.vendorportal.VendorPortalTest" name="logoutTest" time="3.003">
    <skipped/>
  </testcase> <!-- logoutTest -->
  <testcase classname="com.umesh.tests.vendorportal.VendorPortalTest" name="loginTest" time="15.819"/>
  <testcase classname="com.umesh.tests.vendorportal.VendorPortalTest" name="loginTest" time="7.962"/>
  <testcase classname="com.umesh.tests.vendorportal.VendorPortalTest" name="dashboardTest" time="8.112"/>
  <testcase classname="com.umesh.tests.vendorportal.VendorPortalTest" name="logoutTest" time="10.220"/>
  <testcase classname="com.umesh.tests.vendorportal.VendorPortalTest" name="dashboardTest" time="6.062">
    <failure message="expected [8] but found [10]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [8] but found [10]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at com.umesh.tests.vendorportal.VendorPortalTest.dashboardTest(VendorPortalTest.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:481)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- dashboardTest -->
  <testcase classname="com.umesh.tests.vendorportal.VendorPortalTest" name="dashboardTest" time="7.991"/>
</testsuite> <!-- com.umesh.tests.vendorportal.VendorPortalTest -->
